frontend/components/customizer/request-markup-button.tsx

```diff
@@ -3,14 +3,17 @@ import styled from 'styled-components'
 import Select, { components } from 'react-select'
 import { MarkupFormat } from '../../lib/generate-image-markup'
 
-const ClickableControl = (props: any) => (
-  <components.Control
-    {...props}
-    innerProps={{
-      onMouseDown: props.selectProps.onControlMouseDown,
-    }}
-  />
-)
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Why are these inline overrides needed? Is there ruleset conflicts on the function signature, something else?

Reply:
Perhaps we should add a comment. Basically I don't think we should be using `any` or `ts-ignore`, however the correct type of these Control props were difficult to track down and irrelevant to what we're doing here, since we just pass in the ClickableControl class which gets instantiated by react-select.

Reply:
Ahhh okay. I'll confess I've used `ts-ignore` a few times but did feel a little guilty 😄